### ISSUE TITLE
Documentation popup has syntax highlighting issues

### ISSUE THREAD

**Issue by dnicolson** [microsoft/vscode#153938]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.68.1
- OS Version: macOS 12.4

Steps to Reproduce:

1. Hover over `DynamoDBClient` in a TypeScript document

![PixelSnap 2022-07-01 at 19 37 00@2x](https://user-images.githubusercontent.com/2276355/176947386-5b581d7a-1605-4c33-ab22-ace29c68f484.png)

**Comment by dnicolson** [comment]
@mjbvz I was aware of this but isn't there a syntax highlight issue still, the word "and" above for example.

**Comment by mjbvz** [comment]
No, we treat code blocks without an explicit language as belonging to the parent file's language (so they will be rendered as TS in this case)

**Comment by dnicolson** [comment]
I'm wondering what the best solution here is, as the first element is still parsed correctly as long as it isn't a `p` tag (`fullname` or any random tag appear to work). Removing the indentation results in the `p` tags being ignored and removing the `p` tags causes the text to appear again unstyled. The comments look to be correctly parsed in the [TSDoc Playground](https://microsoft.github.io/tsdoc/).

Sublime Text (with [LSP-typescript](https://github.com/sublimelsp/LSP-typescript)) parses the title but not the body:
<img src="https://user-images.githubusercontent.com/2276355/179425101-ac43bcef-1388-41f1-8219-fbc95f7a0e7a.png" width="75%">

Conversely, IntelliJ IDEA parses the `p` tags but not the `fullname` tag:
<img src="https://user-images.githubusercontent.com/2276355/179425104-947b7274-72c4-4e45-8693-6ae8c0b28f93.png" width="75%">

Interestingly, in both Sublime Text and IntelliJ IDEA [single line HTML comments](https://github.com/aws/aws-sdk-js-v3/blob/f3a16cc/clients/client-dynamodb/src/models/models_0.ts#L2031) are parsed correctly but not in Visual Studio Code:
<img src="https://user-images.githubusercontent.com/2276355/179425464-a1b620c3-1076-46e4-b82a-db508e6cd8a8.png" width="75%">

Also, clicking on "Click to show 2 definitions." seems to have no effect.
